charts/minio/1.7.0/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: '2022-11-10'
+appVersion: '2022-10-24'
```

Comment:
Why are we downgrading minio version?

Reply:
Good catch, copied from the test train which looks like haven't been changed there.
Fixed it here, it will be fixed tomorrow on test train too with the bugclerk updates